assets/js/search.js

```diff
@@ -0,0 +1,14 @@
+function toggle_comments(event) {
+    var target = event.target;
+    var body = document.getElementById('filters');
+    if (body.style.display === null || body.style.display === '') {
+        target.innerHTML = '[ + ]';
+        body.style.display = 'none';
+    } else {
+        target.innerHTML = '[ - ]';
+        body.style.display = '';
+    }
+}
+
+document.getElementById('togglefilters').onclick = toggle_comments;
+document.getElementById('filters').style.display = 'none';
```

Comment:
This should be done in CSS to avoid flashing during page load.

Reply:
Hm I wanted to make it work with nojs also.  Should i do some stuff with noscript tag instead for this?

Reply:
Yes, you could have a stylesheet in a `<noscript>` tag.

Reply:
then what's the reason to add this in JavaScript? If it works in CSS (and it can, https://teddit.net does it for comments), I see no reason to code twice the same thing, harder to maintain, harder to track…